app/presenters/openid_connect_user_info_presenter.rb

```diff
@@ -22,7 +22,7 @@ def user_info
     info.merge!(x509_attributes) if scoper.x509_scopes_requested?
     info[:verified_at] = verified_at if scoper.verified_at_requested?
     info[:ial] = Saml::Idp::Constants::AUTHN_CONTEXT_IAL_TO_CLASSREF[identity.ial]
-    info[:aal] = Saml::Idp::Constants::AUTHN_CONTEXT_AAL_TO_CLASSREF[identity.aal]
+    info[:aal] = identity.aal_value
```

Comment:
[question] are we losing some safety that utilizing the constants gave us?

Reply:
Since we are only saving constants in openid_connect_authorize_form.rb I think we are safe here now.